geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-parquet/src/test/scala/org/locationtech/geomesa/parquet/ParquetReadWriteTest.scala

```diff
@@ -88,6 +91,31 @@ class ParquetReadWriteTest extends Specification with AllExpectations {
 
   "SimpleFeatureParquetWriter" should {
 
+    "fail if a corrupt parquet file is written" >> {
+      val filePath = new Path(f.toUri)
+      WithClose(SimpleFeatureParquetWriter.builder(filePath, sftConf).build()) { writer =>
+        features.foreach(writer.write)
+      }
+
+      // Corrupt the file by writing an invalid byte somewhere
+      val randomAccessFile = new RandomAccessFile(f.toFile, "rw")
+      logger.debug(s"File length: ${randomAccessFile.length()}")
+      Files.size(f) must beGreaterThan(50L)
+      randomAccessFile.seek(50)
+      randomAccessFile.writeByte(999)
+      randomAccessFile.close()
+
+      // Set the system property, validate the file, and then unset it
+      FileValidationEnabled.threadLocalValue.set("true")
+      try {
+        validateParquetFile(filePath, logger) must throwA[RuntimeException].like {
+          case e => e.getMessage mustEqual s"Unable to validate ${filePath}: File may be corrupted"
+        }
+      } finally {
+        FileValidationEnabled.threadLocalValue.remove()
```

Comment:
we can remove the threadLocalValue bits here, since they're no longer used/needed.